docs/debugging.md

```diff
@@ -0,0 +1,40 @@
+## Debugging Tutorial
+
+### For debugging purposes the following must be installed :
+
+[Visual Studio Code](https://code.visualstudio.com/)
+[Node.js](https://www.npmjs.com/) - to setup node.js you may follow the instructions in [Environment Setup](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md)
+
+### Setting up VS Code and Launching Telescope :
+
+To begin debugging one must retrieve the launch.json file.
+This file contains **Important** debugging configuration information which should be located in the .vscode folder for telescope to run.
+
+Once the launch.json file has been located, you can move on to the "Run View" tab.  
+[VS Run View Screenshot](https://drive.google.com/file/d/1dvXhen8_eT68qEMa8Z6OpejDiMOG1pLN)
+Visual Studio Code will automatically try to detect the debug environment but in the case that it fails you must select "Node.js".
+[VS Debug Environment Screenshot](https://drive.google.com/file/d/1pvL4RJRJGNLb1LmDZRN3x5UCusxGHdJU/view?usp=sharing)
```

Comment:
Let's add the image to the repo under [`docs/images/`](https://github.com/Seneca-CDOT/telescope/tree/master/docs/images)